networking_oneview/tests/unit/ml2/drivers/oneview/test_oneview_mech_driver.py

```diff
@@ -202,11 +203,39 @@ def test_create_network_postcommit_flat(self, mock_map_net):
             FAKE_FLAT_NETWORK.get('id'),
             mock.ANY, True, ['lig_123', 'uplinkset_flat'])
 
+    @mock.patch.object(database_manager, 'map_neutron_network_to_oneview')
+    def test_create_already_existing_network_postcommit_flat(
+        self, mock_map_net
+    ):
+        network_context = FakeContext()
+        network_context._network = FAKE_FLAT_NETWORK
+        client = self.driver.oneview_client
+        flat_network = {
```

Comment:
I think this can become a final variable at the top of the file. `FAKE_OV_FLAT_NETWORK`